components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/tab/tab-list.js

```diff
@@ -124,7 +124,7 @@ define(['log', 'jquery', 'lodash', 'backbone', './tab', 'bootstrap'], function (
                 tabHeaderLink.text(tab.getTitle());
 
                 tabHeader.setText = function(text){
-                    tabHeaderLink.text(text);
+                    tabHeaderLink.text(text.split(".")[0]);
```

Comment:
single quotes?

Reply:
This is already fixed in the master branch in another way.
https://github.com/wso2/carbon-analytics/blob/f6cb9f072880a6e020db1ff108792560f463f3dd/components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/tab/tab-list.js#L116

Reply:
What I meant was about using single quotes instead of double quotes in js files

Reply:
Yes, I understood. What I actually meant was that my change is no longer there in the code. It is replaced by some other person's change.